### PR TITLE
Add: chromebook usbc fix

### PR DESCRIPTION
**Pull request by Owen-sz**

### CODE REVIEW

File: anda/system/chromebook-usbc-fix/chromebook-usbc-fix.spec [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+%description
+%summary
+
+%prep
+%autosetup -n chromebook-usbc-fix-%commit
+
+%install
+mkdir -p %buildroot%{_sysconfdir}/chromebook-usbc-fix/
+install -Dm755 chromebook-usbc.service %buildroot%{_sysconfdir}/chromebook-usbc-fix/chromebook-usbc.service
```
Comment (madonuko) [inline anda/system/chromebook-usbc-fix/chromebook-usbc-fix.spec]:
```suggestion
install -Dm755 chromebook-usbc.service %buildroot%{_unitdir}/chromebook-usbc.service
```

File: anda/system/chromebook-usbc-fix/chromebook-usbc-fix.spec [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+%postun
+%systemd_postun_with_restart chromebook-usbc.service
+
+%files
+%doc README.md
+%license LICENSE
+%{_sysconfdir}/chromebook-usbc-fix/*
+%{_unitdir}/chromebook-usbc.service
+%{_bindir}/chromebook-usbc-fix
```
Comment (madonuko) [inline anda/system/chromebook-usbc-fix/chromebook-usbc-fix.spec]: This doesn't exist

**Comment by madonuko** [comment]
mock	| DEBUG:     File not found: /builddir/build/BUILD/chromebook-usbc-fix-20241005.cef28b1-build/BUILDROOT/etc/chromebook-usbc-fix/*

**Comment by Owen-sz** [comment]
> mock | DEBUG: File not found: /builddir/build/BUILD/chromebook-usbc-fix-20241005.cef28b1-build/BUILDROOT/etc/chromebook-usbc-fix/*

I saw, don't have time to look into atm

**Comment by madonuko** [comment]
Please mark this as ready for review

**Comment by Owen-sz** [comment]
> Please mark this as ready for review

Waiting for testing

**Review by madonuko — APPROVED** [review]

**Pull request merged**